build/azure-pipeline.pre-release.yml

```diff
@@ -19,6 +19,11 @@ resources:
       ref: main
       endpoint: Monaco
 
+parameters:
+  - name: publishExtension
+    displayName: 🚀 Publish Extension
+    type: boolean
+    default: false
```

Comment:
@lszomoru when this runs automatically on schedule, does this value turn to `true` somehow?

Reply:
Found it https://github.com/microsoft/vscode-engineering/blob/5759362f90e31994d92d8250b8ccb73e1a32a837/azure-pipelines/extension/pre-release.yml#L96